src/static/riot/competitions/editor/_leaderboard.tag

```diff
@@ -261,6 +289,10 @@
             self.show_modal()
         }
 
+        self.change_submission_rule = function (rule) {
+            self.selected_submission_rule = rule
+        }
```

Comment:
Why does this function exist?

Reply:
There's a bug in the modal code that causes the submission rule, and only the submission rule, to be deleted from the DOM when saved. It needs to be recoded when we change leaderboards to phases to a M2M relation anyways. So we can leave this patch here until we need to change it.